### PR TITLE
Improve "tsh kube login" message for proxy behind l7 lb

### PR DESCRIPTION
**Pull request by greedy52**
Part of #27044 and [support-load](https://github.com/gravitational/teleport/labels/support-load)

`tsh kubectl` automatically starts a local proxy in the background (when ALPN upgrade is required) so put in as part of the `tsh kube login` message

Will do a separate PR for doc update

### CODE REVIEW

**Review by capnspacehook — APPROVED** [review]

File: tool/tsh/common/kube.go [hunk 1]
```diff
@@ -1227,14 +1227,24 @@ func (c *kubeLoginCommand) printUserMessage(cf *CLIConf, tc *client.TeleportClie
 func (c *kubeLoginCommand) printLocalProxyUserMessage(cf *CLIConf) {
 	switch {
 	case c.kubeCluster != "":
-		fmt.Fprintf(cf.Stdout(), `Logged into Kubernetes cluster %q. Start the local proxy:
+		fmt.Fprintf(cf.Stdout(), `Logged into Kubernetes cluster %q.
+
+Use 'tsh kubectl' to run the Kubernetes client, for example:
```
Comment (tigrato) [inline tool/tsh/common/kube.go:1232]: Can we give more detail here saying: hey look tsh kubectl is awesome and it doesn't require a local proxy :laughing:

Comment (greedy52) [reply tool/tsh/common/kube.go:1232]: i am really bad at this, any suggestions?

Comment (tigrato) [reply tool/tsh/common/kube.go:1232]:
```
Logged into Kubernetes cluster %q.

Your cluster runs behind an L7 load balancer and to access the cluster, you can use:

- 'tsh kubectl'  - a fully featured `kubectl` command that works when the cluster is behind L7 LB; To run the Kubernetes client, use:
     tsh kubectl version

- Start a local proxy 
    tsh proxy kube -p 8443
   Configure the kubeconfig provided by the local proxy, and test the connection with:
   'kubectl version'

Learn more at https://goteleport.com...
```

What do you think?
Maybe adding a link to more elaborated documentation would be a helper.

Comment (greedy52) [reply tool/tsh/common/kube.go]: I didn't find any message tsh prints with `-` lists. How does this look?
```
Logged into Kubernetes cluster "<kube-cluster>".

Your Teleport cluster runs behind a layer 7 load balancer or reverse proxy.

To access the cluster, use "tsh kubectl" which is a fully featured "kubectl"
command that works when the Teleport cluster is behind layer 7 load balancer or
reverse proxy. To run the Kubernetes client, use:
  tsh kubectl version

Or, start a local proxy with "tsh proxy kube" and use the kubeconfig
provided by the local proxy with your native Kubernetes clients:
  tsh proxy kube -p 8443

Learn more at https://goteleport.com/docs/architecture/tls-routing/#working-with-layer-7-load-balancers-or-reverse-proxies-preview
```

**Review by smallinsky — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

**Pull request merged**